src/sentry/event_manager.py

```diff
@@ -110,17 +115,23 @@ def get_hashes_from_fingerprint(event, fingerprint):
     for idx in range(hash_count):
         result = []
         for bit in fingerprint:
-            if bit in default_values:
+            if bit in DEFAULT_FINGERPRINT_VALUES:
                 result.extend(default_hashes[idx])
             else:
                 result.append(bit)
         hashes.append(result)
     return hashes
 
 
+def get_hashes_from_fingerprint(event, fingerprint):
+    return _get_hashes_from_fingerprint(
+        functools.partial(get_hashes_for_event, event),
+        fingerprint,
+    )
+
+
 def get_hashes_from_fingerprint_with_reason(event, fingerprint):
```

Comment:
Not necessarily relevant to this specific patch, but seems like we really should consolidate this with the above implementation in the future.